test/system/practice/completion_test.rb

```diff
@@ -8,6 +8,14 @@ class Practice::CompletionTest < ApplicationSystemTestCase
     assert_text '「OS X Mountain Lionをクリーンインストールする」'
   end
 
+  test 'Appropriate meta description is displayed when accessed by non-logged-in user' do
```

Comment:
```suggestion
  test 'appropriate meta description is displayed when accessed by non-logged-in user' do
```
他のテストに合わせて小文字から始めるのがいいと思います〜